packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -460,19 +460,6 @@ export interface IFluidParentContext
 	 */
 	getAudience(): IAudience;
 
-	/**
-	 * Invokes the given callback and expects that no ops are submitted
-	 * until execution finishes. If an op is submitted, an error will be raised.
-	 *
-	 * Can be disabled by feature gate `Fluid.ContainerRuntime.DisableOpReentryCheck`
-	 *
-	 * @param callback - the callback to be invoked
-	 *
-	 * @deprecated
-	 * // back-compat: to be removed in 2.0
-	 */
-	ensureNoDataModelChanges<T>(callback: () => T): T;
```

Comment:
This is one you shouldn't remove yet, since it's not `@internal`

Reply:
Oh... the other ones depend on this so you can't remove them yet either.  Too bad.